src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/NotifyChangeFunction.cs

```diff
@@ -48,25 +48,46 @@ public class NotifyChangeFunction
         {
             logger.LogInformation($"{executionContext.FunctionName} triggered: {message}");
             await MarkScheduledReleaseStatusAsSuperseded(message);
-            await ValidateReleaseAsync(message, async () =>
+            if (! await ValidatePublishingState(message, logger))
```

Comment:
If `ValidatePublishingState` returns false, wouldn't we want to do nothing? Seems like we would be continuing as normal if it's false, which seems weird.

Reply:
Yep, should have been `if (await ValidatePublishingState` rather than `if (! await ValidatePublishingState`. I've corrected it now.